Type error when running ng test

**Describe the bug**
```
ERROR in src/typings.d.ts:7:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'window' must be of type 'Window & typeof globalThis', but here has type 'Window'.

7 declare var window: Window;
              ~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:19545:13
    19545 declare var window: Window & typeof globalThis;
                      ~~~~~~
    'window' was also declared here.
```

--- a/src/typings.d.ts
+++ b/src/typings.d.ts
@@ -4,7 +4,7 @@ interface NodeModule {
   id: string;
 }
 
-declare var window: Window;
+declare var window: Window & typeof globalThis;
 interface Window {
   process: any;
   require: any;

